link_shortener/commands/filter.py

```diff
@@ -0,0 +1,17 @@
+'''
+Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
+Licensed under the MIT (Expat) License (see LICENSE in Documentation).
+'''
+
+
+async def filter_links(link_data, filters):
```

Comment:
Jo tohle se mi líbí, jen to potřebuju bez konfliktů abych to k sobě mohl natáhnout